rust/stracciatella/src/schemas/yaml/explosives.schema.yaml

```diff
@@ -16,6 +16,10 @@ items:
           title: Animation ID
           description: ID of the explosion animation used for this explosive
           $ref: types/id.schema.yaml
+        calibre:
+          title: Calibre
+          description: The calibre of the launcher weapon that must be used to launch this explosive.
+          $ref: types/id.schema.yaml
```

Comment:
Shouldn't this use the new schema?

Reply:
As it's just a reference to the new schema, this is just a string. The calibres are their own file.